Client/Frontend/Browser/BrowserViewController/BrowserViewController+Callout.swift

```diff
@@ -31,6 +31,40 @@ extension BrowserViewController {
     }
   }
   
+  func presentBottomBarCallout() {
+    if Preferences.DebugFlag.skipNTPCallouts == true || isOnboardingOrFullScreenCalloutPresented { return }
+
+    if presentedViewController != nil || !FullScreenCalloutManager.shouldShowDefaultBrowserCallout(calloutType: .bottomBar) {
+      return
+    }
+
+    let onboardingNotCompleted =
+      Preferences.Onboarding.basicOnboardingCompleted.value != OnboardingState.completed.rawValue
+
+    if onboardingNotCompleted || !Preferences.General.isUsingBottomBar.value {
```

Comment:
`!Preferences.General.isUsingBottomBar.value` should not this be reversed? We want to show it only for existing users who do not have it enabled

Reply:
Fixed and separated https://github.com/brave/brave-ios/pull/6640/commits/1b9470aca9072e5e1bcdfd337345da622ab950d1